css-scroll-anchoring-1/Overview.bs

```diff
@@ -81,7 +81,7 @@ and close to the block start edge of its <a>optimal viewing region</a>.
 Note: If the user agent does not support the 'scroll-padding' property,
 the optimal viewing region of the scrolling box is equivalent to its <a>content area</a>.
 
-The anchor node is either a non-anonymous <a>block box</a> or a <a>text node</a>.
+An anchor node can be any non-<a>atomic inline</a> <a>box</a>.
```

Comment:
I think this reads a bit weird, but it may be because I'm not a native English speaker. When I see this, it kinda wants to read "Any inline box which is non-atomic", which is exactly the opposite of what we want. I guess in the spec it would be a bit clearer because `atomic inline` is a concept on its own, but maybe this should read "`any box except <a>inline boxes</a> that are non-<a for="atomic inline">atomic</a>`" or such (whatever makes bikeshed generate the right markup)?

Reply:
Adjusted to make it clearer.